dev-requirements.txt

```diff
@@ -45,16 +45,20 @@ coveralls>=2.1.2,<3.0.0; python_version >= '3.5'
 # Safety CI by pyup.io
 # safety 1.9.0 removed support for Python 2.7 (and now also enforces that)
 safety>=1.8.7,<1.9.0; python_version == '2.7'
-safety>=1.9.0; python_version >= '3.5'
+safety>=1.9.0,<2.0.0; python_version == '3.5'
+safety>=2.2.0; python_version >= '3.6'
 # dparse 0.5.0 has an infinite recursion issue on Python 2.7,
 #   see https://github.com/pyupio/dparse/issues/46
 dparse>=0.4.1,<0.5.0; python_version == '2.7'
-dparse>=0.5.2; python_version >= '3.5'
-# Safety requires Click>6.0 and the upgrade strategy 'eager' causes Click to be
-# upgraded to 8.0.0, unless we repeat the Click requirements from
-# requirements.txt. Keep them in sync.
+dparse>=0.5.2; python_version == '3.5'
+# ver 0.6.2 min requirement by safety 2.2.0
+dparse>=0.6.2; python_version >= '3.6'
+# Safety requires Click>=8.0.2.
+# Repeat the Click requirements from
+# requirements.txt. to keep them in sync.
 Click>=7.1.1,<8.0; python_version <= '3.5'
-Click>=8.0.1; python_version >= '3.6'
+# Safety package requires click 8.0.2 minimum
+# Click>=8.0.2; python_version >= '3.6'
```

Comment:
Given that we include the requirements.txt file, we should not repeat Click here since it is specified there.